src/GrpcProtos/capture.proto

```diff
@@ -27,6 +27,13 @@ message InstrumentedFunction {
   string function_name = 5;
 }
 
+message ApiTableInfo {
+  string module_path = 1;
+  uint64 api_table_address = 2;
```

Comment:
is this absolute address? maybe we need to pass it in form of file_path and offset, the way it is done for instrumented functions and have the service side to figure out where the module is loaded?

Also this will let us do it on the fly without involvement of the client if a module is not yet loaded.

Reply:
Changed so that we only send the relative address, module_path and build_id. The service verifies the build-id's match and computes the absolute address.